src/paths.py

```diff
@@ -41,7 +45,7 @@ def lookupAppdataFolder():
             dataFolder = os.path.join(
                 os.environ['HOME'],
                 'Library/Application Support/', APPNAME
-            ) + '/'  # FIXME: should also be os.path.sep
+            ) + '/'  # ..fixme:: should also be os.path.sep
```

Comment:
maybe it should be `os.path.sep`

Reply:
@lakshyacis I cannot find such directive here: https://www.sphinx-doc.org/en/1.8/usage/restructuredtext/index.html
@PeterSurda I intentionally didn't replace because in that case I had to test it

Reply:
No such reST directive `..fixme:`